tool/tsh/tsh.go

```diff
@@ -1662,7 +1666,11 @@ func reissueWithRequests(cf *CLIConf, tc *client.TeleportClient, reqIDs ...strin
 	if err := tc.SaveProfile("", true); err != nil {
 		return trace.Wrap(err)
 	}
-	if err := kubeconfig.UpdateWithClient(cf.Context, "", tc, os.Args[0]); err != nil {
+	tshPath, err := os.Executable()
```

Comment:
I know this might be frowned up but given the number of uses for `os.Executable` through the codebase, I would store the path to the executable in a global variable (which might be a legitimate use for it) during init and read it w/o having to handle errors.

Reply:
good idea, added it to `CLIConf` which is the global config struct for all commands.